pages/content/navigation.mdx

```diff
@@ -0,0 +1,106 @@
+import { Callout } from "nextra/components";
+import { Box, Text, Section } from "@radix-ui/themes";
+
+# Navigation
+
+Navigation is configured from `_meta.json` files located in `/content`.
+
+## Primary Navigation
+
+Canopy IIIF creates it's primary navigation at build time from `/content/_meta_.json`. Each link will have a `path` and `text` key. A `path` value will resolve to a file or directory under `/content` or `/src/pages`. The `text` key will be what's displayed in the navigation bar. The order of the links in the array will be the order they appear in the navigation.
+
+```json filename="/content/_meta_.json"
+[
+  {
+    "path": "/works",
+    "text": "Works"
+  },
+  {
+    "path": "/metadata",
+    "text": "Metadata"
+  },
+  {
+    "path": "/about",
+    "text": "About"
+  }
+]
+```
+
+<Box m="9" my="6">
+  ![canopy home page](/examples/content-navigation-primary.png)
+</Box>
+
+<Callout type="info">
+  Canopy IIIF provides the **Work** and **Metadata** pages out of the box,
+  pre-configured. Therefore you will not see markdown files for these pages
+  under the `/content` directory, though their links will resolve.
+</Callout>
+
+For those looking for a more detailed explanation of how the `path` value is resolved, see below, otherwise skip the Technical Explanation below.
+
+<Callout>
+  **Technical explanation**: The `path` value follows
+  [NextJS's](https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts)
+  `pages` routing convention. The `path` value `/works` will map to
+  `/src/pages/works/index.tsx`. The `path` value`/metadata` will map to
+  `/src/pages/metadata.tsx`.
+</Callout>
+
+### Custom Content
+
+Most users will want to add navigation for custom content to their site. For example, the **About** page, or creating a new page. This is done by creating a markdown file under `/content` and adding a link to the primary navigation (as seen above). For example, to add a page called **About** to the primary navigation, create a markdown file called `index.mdx` under `/content/about`.
+
+<Callout type="info">
+  If creating a new page, try copying the content from `src/pages/about` into
+  `src/pages/{your_page_name}`. This will give you a good starting point for
+  your new page.
+</Callout>
+
+## Secondary Navigation
+
+### Top Level
+
+Secondary navigation may be defined from within a `_meta.json` file located within a subdirectory of `/content`. Examining the **About** example...
```

Comment:
Top Level seems to contrast to much with Secondary. What if we call this Content Navigation? Also, a little beef.

```suggestion
### Content Navigation

An example directory `content/about` is included in a new Canopy project. Implementers may want to modify navigation to match their custom content. This directory has its own `_meta.json` file that defines navigation at this level.
```